site/content/en/reference/live/install-resource-group.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Install-resource-group"
```

Comment:
In doc titles and description, mark this as `alpha` as well? It's only relevant for the alpha feature being added. Same for migrate.

Reply:
Fixed